src/plugin/filemodel.cpp

```diff
@@ -489,7 +483,7 @@ void FileModel::readAllEntries()
         return;
     }
 
-    if (euidaccess(dir.path(), R_OK) == -1) {
+    if (faccessat(AT_FDCWD, dir.path().toStdString().c_str(), R_OK, AT_EACCESS) == -1) {
```

Comment:
Very nice and small fix :+1: .

I'm not 100% sure about std::string -> c_str and what happens to ä,ö,ü,ß, etc. Hence, I'd rather stick with previous toUtf8() + constData() (constData() instead of data()). 
```
QByteArray pathBytes = dir.path().toUtf8();
const char *pathName = pathBytes.constData();
if (faccessat(AT_FDCWD, pathBytes, R_OK, AT_EACCESS) == -1) {
```

Add local variable for QByteArray to avoid loosing referring to already destroyed data.